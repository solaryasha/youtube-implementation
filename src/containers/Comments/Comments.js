import React, { Component } from 'react';
import './Comments.scss';
import CommentsHeader from './CommentsHeader/CommentsHeader';
import AddComment from './AddComment/AddComment';
import Comment from './Comment/Comment';

export class Comments extends Component {
  render() {
    return (
      <div>
        <CommentsHeader commentsAmount={this.props.commentsAmount} />
        <AddComment />
        <Comment />
        <Comment />
        <Comment />
        <Comment />
        <Comment />
      </div>
    );
  }
}
