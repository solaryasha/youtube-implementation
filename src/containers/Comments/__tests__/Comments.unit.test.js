import React from 'react';
import { shallow } from 'enzyme';
import { Comments } from '../Comments';


describe('Comments', () => {
    test('renders without props', () => {
        const wrapper = shallow(<Comments/>);

        expect(wrapper).toMatchSnapshot();
    })

    test('renders with commentsAmount', () => {
        const wrapper = shallow(<Comments commentsAmount={505050}/>);

        expect(wrapper).toMatchSnapshot();
    })
})