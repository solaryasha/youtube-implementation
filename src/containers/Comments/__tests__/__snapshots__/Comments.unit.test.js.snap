// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders with commentsAmount 1`] = `
<div>
  <CommentsHeader
    commentsAmount={505050}
  />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;

exports[`Comments renders without props 1`] = `
<div>
  <CommentsHeader />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;
